src/tools/crossgen2/Common/Compiler/DependencyAnalysis/Relocation.cs

```diff
@@ -25,6 +25,11 @@ public enum RelocType
         IMAGE_REL_BASED_ARM64_PAGEBASE_REL21 = 0x81,   // ADRP
         IMAGE_REL_BASED_ARM64_PAGEOFFSET_12A = 0x82,   // ADD/ADDS (immediate) with zero shift, for page offset
         IMAGE_REL_BASED_ARM64_PAGEOFFSET_12L = 0x83,   // LDR (indexed, unsigned immediate), for page offset
+
+        //
+        // Relocations for R2R image production
+        //
+        IMAGE_REL_SYMBOL_SIZE = 0x1000,             // The size of data in the image representated by the target symbol node
```

Comment:
Nit - please teach me as a non-native English speaker - representated or represented?

Reply:
represented :) Good catch

Reply:
I think I tried to invent a new word. Fixed.